templates/all-services/all-services-en.html

```diff
@@ -49,755 +49,103 @@ <h2 class="h3 mb-4">Most requested services</h2>
-							<li><a href="https://www.canada.ca/en/services/policing/corrections.html">Corrections</a></li>
-							<li><a href="https://www.canada.ca/en/services/policing/parole.html">Parole, record suspension, expungement and clemency</a></li>
-							<li><a href="https://www.canada.ca/en/services/policing/victims.html">Victims of crime</a></li>
-						</ul>
-					</div>
-				</div>
-			</details>
-		</section>
-		<section>
-			<details>
-				<summary>
-					<h2 class="h5 fnt-nrml">Science and innovation</h2>
-				</summary>
-				<div class="bg-light row pt-3 pb-2">
-					<div class="col-md-3">
-						<h3 class="mt-0">Most requested</h3>
-					</div>
-					<div class="col-md-9">
-						<ul class="colcount-md-2 compact bold-content lst-spcd fnt-hdng">
-							<li><a href="https://nrc.canada.ca/en/certifications-evaluations-standards/codes-canada/codes-canada-publications">National building codes</a></li>
-							<li><a href="https://nrc.canada.ca/en/web-clock/">Official times across Canada</a></li>
-							<li><a href="https://nrc.canada.ca/en/research-development/products-services/software-applications/sun-calculator/">Check sunrise and sunset times</a></li>
-							<li><a href="https://nrc.canada.ca/en/support-technology-innovation/financial-support-technology-innovation-through-nrc-irap">Grants for technological innovation (IRAP)</a></li>
-							<li><a href="https://science-libraries.canada.ca/eng/home/">Federal Science Libraries Network</a></li>
-							<li><a href="https://www.asc-csa.gc.ca/eng/astronomy/northern-lights/auroramax-observatory.asp">Live view of northern lights cam</a></li>
-						</ul>
-					</div>
-				</div>
-				<div class="row pt-3 pb-2">
-					<div class="col-md-3">
-						<h3 class="mt-0">More topics</h3>
-					</div>
-					<div class="col-md-9">
-						<ul class="colcount-md-2 compact bold-content lst-spcd fnt-hdng">
-							<li><a href="https://www.canada.ca/en/services/science/researchfunding.html">Research funding and awards</a></li>
-							<li><a href="https://www.canada.ca/en/services/science/sciencesubjects.html">Science subjects</a></li>
-							<li><a href="https://www.canada.ca/en/services/science/open-data.html">Open data, statistics and archives</a></li>
-							<li><a href="https://www.canada.ca/en/services/science/institutes.html">Research institutes and facilities</a></li>
-							<li><a href="https://www.canada.ca/en/services/science/innovation.html">R&D and innovation</a></li>
-							<li><a href="https://www.canada.ca/en/services/business/ip.html">Intellectual property and copyright</a></li>
-							<li><a href="https://www.canada.ca/en/services/science/scientistsdirectory.html">Directory of scientists and research professionals</a></li>
-						</ul>
-					</div>
-				</div>
-			</details>
-		</section>
+</section>
+</div>
+<div class="container">
+<h2>Focus on</h2>
+<section class="gc-srvinfo">
+	<div class="wb-eqht row">
+		<div class="col-md-6 col-lg-4">
```

Comment:
The section is missing a single top level heading,

Move the "h2" inside and leverage the equal row CSS instead.

```suggestion
<section class="gc-srvinfo">
	<h2>Focus on</h2>
	<div class="wb-eqht-grd row">
		<div class="col-md-6 col-lg-4">
```